### ISSUE TITLE
How can I make text-fragment feature work on pdfjs

### ISSUE THREAD

**Issue by pkgitlab** [mozilla/pdf.js#15223]
Greetings pdfjs experts,

Are there any plans to support https://wicg.github.io/scroll-to-text-fragment/ ?  We would like to create urls with pre-selected texts to specific page of a pdf file(which uses a #).  

For example this can be one scenario: https://mozilla.github.io/pdf.js/web/viewer.html#:~:text=Dynamic

Or for example if i want to prehighlight TraceMonkey here
https://mozilla.github.io/pdf.js/web/viewer.html#page=5

Thank you very much, this software has given a new life to our project.
Prashant

**Comment by Snuffleupagus** [comment]
> Are there any plans to support https://wicg.github.io/scroll-to-text-fragment/ ? We would like to create urls with pre-selected texts to specific page of a pdf file(which uses a #).

No, since we already (mostly) support the hash-parameters that exist *specifically* for PDF viewers.

> For example this can be one scenario: https://mozilla.github.io/pdf.js/web/viewer.html#:~:text=Dynamic

The following already works: https://mozilla.github.io/pdf.js/web/viewer.html#search=Dynamic

> Or for example if i want to prehighlight TraceMonkey here
> https://mozilla.github.io/pdf.js/web/viewer.html#page=5

Similar to the above: https://mozilla.github.io/pdf.js/web/viewer.html#page=5&search=TraceMonkey

**Comment by pkgitlab** [comment]
Thanks for quick response, my request was because Google has started to append text-fragment to url on several website and we were wondering what would happen if we allowed Google to index our pdfs page by page (dynamically generate pages during their crawl).   But what you said is helpful too!
